common/OpTestSSH.py

```diff
@@ -36,9 +36,10 @@ class ConsoleState():
     DISCONNECTED = 0
     CONNECTED = 1
 
-def set_system_to_UNKNOWN(system):
+def set_system_to_OFF(system):
```

Comment:
As we discussed in slack, setting to OFF will break OPexpect wrapper capturing console error messages. so please fix this by calling failure_callback either just before or just after we raise the assertion/exceptions for kernel failures.